### PR TITLE
Cleanup for JIRA 2095

### PR DESCRIPTION
**Pull request by sbroeder**

### CODE REVIEW

**Comment by sbroeder** [comment]
Jenkins, retest

**Comment by DaveBirdsall** [comment]
Looks good to me.

**Comment by sbroeder** [comment]
Jenkins, retest

File: core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java [hunk 1]
```diff
@@ -1484,23 +1484,18 @@ public PushEpochResponse call(TrxRegionService instance) throws IOException {
               + new String(startKey, "UTF-8") + " endKey: " + new String(endKey, "UTF-8"));
 
        boolean loopExit = false;
```
Comment (selvaganesang) [inline core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/TransactionManager.java]: Change looks good. Please consider removing this variable when you have a chance.

**Comment by DaveBirdsall** [comment]
jenkins, retest

**Pull request merged**